Prototype Emote Component

347Online opened the pull request:
Created basis of React App, and the Prototype Emote component
Added style.css
Small tweak to Webpack, was erroring on CSS imports
Edited README.md with the current version of the Front End <-> Back End Protocol

347Online committed:
Prototype Emote Component

Created basis of React App, and the Prototype Emote component
Added style.css
Small tweak to Webpack, was erroring on CSS imports
Edited README.md with the current version of the Front End <-> Back End Protocol

EZW1 reviewed (changes requested):
Great job! 2 things:

- Make sure to use unique keys when iterating to render sibling components (possible use message _id key?)
- Consider making a comment to indicate where App.jsx is being rendered, as it should ber rendered in index.js

Thanks Katie!

347Online committed:
Requested Changes

message _id is not an available property currently, will switch the React Component 'key' when it becomes available
Added comment clarifying that the App is attached to the DOM from within App.jsx
